Automattic/wc-calypso-bridge
# Remove LYS badge from WPCOM sites

### Changes proposed in this Pull Request:

p1727162923759179/1726818329.636839-slack-C06DJHTE04U

Since WPCOM has coming soon badge shown in sidebar when in coming soon mode, LYS coming soon badge is redundant.

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/127d78d7-b3b9-47bb-9f38-1e5a54a0c033">


### How to test the changes in this Pull Request:

1. Use a WPCOM atomic e-commerce site
1. Enable LYS feature flag (you can set [these lines](https://github.com/Automattic/wc-calypso-bridge/blob/507bde7b933c60bb2cb7efdfad66b7bd8581b39b/includes/class-wc-calypso-bridge-woocommerce-admin-features.php#L143-L147) to `true`).
1. Go to `Settings > WooCommerce > Site visibility` 
2. Set to `Coming soon` and save
1. Confirm site visibility badge is not shown



### Other information:




### FOR PR REVIEWER ONLY:
